tmt/utils.py

```diff
@@ -6183,3 +6183,29 @@ def __exit__(self, *args: Any) -> None:
     @property
     def duration(self) -> datetime.timedelta:
         return self.end_time - self.start_time
+
+
+def retry(
+        function_to_retry: Callable[..., Any],
+        retry_count: int,
+        delay: int,
```

Comment:
* `function_to_retry` is a name a bit too verbose, quite common names used for "code to call" are `callable`, `fn`, or `func`.
* `retry_count` - I'd recommend `retries` instead, that's, again, a common name for "how many times shall we try again" variables.